datasets/ohsumed/README.md

```diff
@@ -0,0 +1,205 @@
+---
+annotations_creators:
+- human-annotated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-nc-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 100k< n<500K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- multi-label-classification
```

Comment:
What are the classification input/target?

Reply:
Pls check my note on the supervised task below.